liquidctl/cli.py

```diff
@@ -11,43 +11,43 @@
+  -m, --match <substring>       Filter devices by description substring
+  -n, --pick <number>           Pick among many results for a given filter
+  --vendor <id>                 Filter devices by hexadecimal vendor ID
+  --product <id>                Filter devices by hexadecimal product ID
+  --release <number>            Filter devices by hexadecimal release number
+  --serial <number>             Filter devices by serial number
+  --bus <bus>                   Filter devices by bus
+  --address <address>           Filter devices by address in bus
+  --usb-port <port>             Filter devices by USB port in bus
+  -d, --device <id>             Select device by listing id
 
 Animation options (devices/modes can support zero or more):
-  --speed <value>             Abstract animation speed (device/mode specific)
-  --time-per-color <value>    Time to wait on each color (seconds)
-  --time-off <value>          Time to wait with the LED turned off (seconds)
-  --alert-threshold <number>  Threshold temperature for a visual alert (°C)
-  --alert-color <color>       Color used by the visual high temperature alert
-  --direction <string>        If the pattern should move forward or backwords. [default: forward]
-  --start-led <number>        The first led to start the effect at (0 offset)
-  --num-leds <number>         The number of LED's the effect shoud apply to
+  --speed <value>               Abstract animation speed (device/mode specific)
+  --time-per-color <value>      Time to wait on each color (seconds)
+  --time-off <value>            Time to wait with the LED turned off (seconds)
+  --alert-threshold <number>    Threshold temperature for a visual alert (°C)
+  --alert-color <color>         Color used by the visual high temperature alert
+  --direction <string>          If the pattern should move forward or backwards. [default: forward]
+  --start-led <number>          The first led to start the effect at
+  --maximum-leds <number>       The number of LED's the effect should apply to
 
 Other device options:
-  --single-12v-ocp            Enable single rail +12V OCP
-  --pump-mode <mode>          Set the pump mode (certain Corsair coolers)
-  --legacy-690lc              Use Asetek 690LC in legacy mode (old Krakens)
-  --non-volatile              Store on non-volatile controller memory
-  --unsafe <features>         Comma-separated bleeding-edge features to enable
-  --temp-sensor <number>      The tempature sensor number for the Commander Pro 
+  --single-12v-ocp              Enable single rail +12V OCP
+  --pump-mode <mode>            Set the pump mode (certain Corsair coolers)
+  --legacy-690lc                Use Asetek 690LC in legacy mode (old Krakens)
+  --non-volatile                Store on non-volatile controller memory
+  --unsafe <features>           Comma-separated bleeding-edge features to enable
+  --temperature-sensor <number> The temperature sensor number for the Commander Pro
```

Comment:
docopt requires two spaces between an option and its description.   Even if this still somehow parses (correctly), let's follow the spec.